common/logisticspipes/gui/GuiGuideBook.java

```diff
@@ -0,0 +1,316 @@
+package logisticspipes.gui;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.GuiButton;
+import net.minecraft.client.gui.GuiScreen;
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.util.EnumHand;
+import net.minecraft.util.ResourceLocation;
+
+import org.lwjgl.opengl.GL11;
+
+import java.util.Arrays;
+
+import logisticspipes.LPConstants;
+import logisticspipes.items.ItemGuideBook;
+import logisticspipes.utils.string.StringUtils;
+
+public class GuiGuideBook extends GuiScreen {
+
+	/**
+	 * Z Levels:
+	 * Title and Buttons: 15
+	 * Background frame : 10
+	 * Text             : 5
+	 * Background       : 0
+	 * Background tint  : -5
+	 */
+
+	private static final ResourceLocation GUI_BOOK_TEXTURE = new ResourceLocation(LPConstants.LP_MOD_ID, "textures/gui/guide_book.png");
+
+	private GuiButton nextPageBtn, prevPageBtn, slider;
+
+	private int mouseX, mouseY;
+	private ItemStack book;
+	private EnumHand hand;
+
+	//Slider vars
+	private boolean dragging = false;
+	private boolean draggable = false;
+	private float sliderProgress;
+	private int sliderTopY;
+	private int sliderBotY;
+	private final int sliderWidth = 20, sliderHeight = 20;
+
+	//Book vars
+	private int page;
+	private int pageMax = 11;
+	private int maxLength;
+	private int textX, textY;
+	private int maxLines;
+
+	//Text vars
+	private List<String> text = new ArrayList<>();
+
+	//Drawing Constants
+	private final float textureSide$x = 256.0F;
+	private final float textureSide$y = 256.0F;
```

Comment:
Unless you plan on adding more stuff to that texture, shrink it (I think 128x64 should still fit everything)

Reply:
I think I'll keep it as is just because I don't know how much space I will need for the next step I'm thinking of taking 👍